pyquil/experiment/_result.py

```diff
@@ -33,6 +35,37 @@
 log = logging.getLogger(__name__)
 
 
+def bitstrings_to_expectations(
+        bitstrings: np.ndarray,
+        correlations: Optional[List[List[int]]] = None
+) -> np.ndarray:
+    """
+    Given a list of bitstrings (each of which is represented as a list of bits), map them to
+    expectation values and return the desired correlations. If no correlations are given, then just
+    the 1 -> -1, 0 -> 1 mapping is performed.
+
+    :param bitstrings: List of bitstrings to map.
+    :param correlations: Correlations to calculate. Defaults to None, which is equivalent to the
```

Comment:
I can't tell from the documentation what this correlations stuff is about. Bitstrings are specified as an array, correlations are a list of lists. The default is `[[0], [1], ..., [n-1]]`, whatever that means. Can we add a bit more here, or above, on the meaning of this?